pages/common/git-count.md

```diff
@@ -0,0 +1,13 @@
+# git count
+
+> Print the total number of commits.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-count>.
+
+- Print the total number of commits:
+
+`git count`
+
+- Print the total number of commits and the number of commits per contributor:
```

Comment:
```suggestion
- Print the number of commits per contributor and the total number of commits:
```
It's really unsatisfying when the order of the description doesn't match the output 😄